src/collector.ts

```diff
@@ -80,7 +80,7 @@ export const collect = async (inUrl: string, args: CollectorOptions) => {
             },
             node_version: process.version
         },
-        start_time: new Date(),
+        start_time: new Date().toISOString(),
```

Comment:
In respect to the API, this seems like a reasonable change. However, for a package, I think it would be more useful for any developers using it to have the full date object to do with as they please. If we ever change the package to return instead of writing to a directory, I think we should change it back.

Reply:
I agree. As a developer I would like it to be an object. However, if you check this doc for API design, it said:
scan_time: Date and time the page was scanned. (String, ISO8610 extended. e.g. 2023-06-06T15:42:28Z. Always UTC.)

I am following that doc to redesign the API fields. If just for the purpose of displaying on the graphic then string is good. But as you said, it is a package so people might want to use the object instead. We might want to ask Simon's opinion since he wrote the API design doc.

Reply:
Right, but collector != API